app/src/main/java/ar/com/wolox/android/example/model/SupportedYouTubeObject.kt

```diff
@@ -0,0 +1,8 @@
+package ar.com.wolox.android.example.model
+
+import java.io.Serializable
+
+/**
+ * Model to wrap a [com.google.api.services.youtube.model.SearchResult] to pass ids as extras
+ */
+data class SupportedYouTubeObject(val kind: String, val playlistId: String?, val videoId: String?) : Serializable
```

Comment:
I don't like this name, this doesn't express what this object is for. I think something like `SearchResultAdapter` could be better, but feel free to think another 😄

Reply:
This model works as a `com.google.api.services.youtube.model.SearchResult` subset to pass only supported `YouTube API` responses, I'll name it as `SearchItemAdapter`.

Reply:
As the class is a model, I find `Adapter` not appropriate, so I think I'll name that model as `SearchResultSubSet` to give it more sense.